### PR TITLE
fix: show divide warning only when dividing

### PR DESCRIPTION
**Pull request by yasmin-e**
Closes: [614](https://github.com/lightdash/lightdash/issues/614) 

### Description:
Currently, we show the following warning to all postgres/redshift users.

The warning takes up a lot of screen space in the modal. It would be much better to show this conditional on whether a divide / has actually been used in the table calculation. The warning is not informative if you're not actually dividing.

image

### CODE REVIEW

File: packages/frontend/src/components/TableCalculationModels/TableCalculationModal.tsx [hunk 1]
```diff
@@ -120,14 +121,19 @@ const TableCalculationModal: FC<Props> = ({
...
-                            ),
-                            displayName: name,
-                            sql,
-                        });
+                        // eslint-disable-next-line @typescript-eslint/no-unused-expressions
+                        sql?.includes('/') &&
+                        (targetDatabase === 'postgres' ||
+                            targetDatabase === 'redshift')
+                            ? setIsDividing(true)
+                            : onSave({
```
Comment (IrakliJani) [inline packages/frontend/src/components/TableCalculationModels/TableCalculationModal.tsx:129]: will the onSave ever happen if input has `/` in the content?

Comment (yasmin-e) [reply packages/frontend/src/components/TableCalculationModels/TableCalculationModal.tsx:129]: nope, should it ?

**Comment by rephus** [comment]
Migration didn't run correctly on render, I'll fix it

File: packages/frontend/src/components/TableCalculationModels/TableCalculationModal.tsx (hunk 1 above)
Comment (rephus) [reply packages/frontend/src/components/TableCalculationModels/TableCalculationModal.tsx:129]: @IrakliJani is right, I can't save the TC if / is present 

[Screencast from 23-12-22 16:32:49.webm](https://user-images.githubusercontent.com/1983672/209360853-2bff01c8-a221-47ce-9ac0-8f82b9a7ffda.webm)

**Review by owlas — APPROVED** [review]

**Review by IrakliJani — APPROVED** [review]

**Pull request merged**